test/suites/config.sh

```diff
@@ -287,6 +287,72 @@ test_config_edit() {
     lxc delete foo
 }
 
+test_property() {
+  ensure_import_testimage
+
+  lxc init testimage foo -s "lxdtest-$(basename "${LXD_DIR}")"
+
+  # Set a property of an instance
+  lxc config set foo description="a new description" --property
+  # Check that the property is set
+  lxc config show foo | grep -q "description: a new description"
+
+  # Unset a property of an instance
+  lxc config unset foo description --property
+  # Check that the property is unset
+  ! lxc config show foo | grep -q "description: a new description" || false
+
+  # Set a property of an instance (bool)
+  lxc config set foo ephemeral=true --property
+  # Check that the property is set
+  lxc config show foo | grep -q "ephemeral: true"
+
+  # Unset a property of an instance (bool)
+  lxc config unset foo ephemeral --property
+  # Check that the property is unset (i.e false)
+  lxc config show foo | grep -q "ephemeral: false"
+
+  # Create a snap of the instance to set its expiration timestamp
+  lxc snapshot foo s1
+  lxc config set foo/s1 expires_at="2024-03-23T17:38:37.753398689-04:00" --property
+  lxc config show foo/s1 | grep -q "expires_at: 2024-03-23T17:38:37.753398689-04:00"
+  lxc config unset foo/s1 expires_at --property
+  lxc config show foo/s1 | grep -q "expires_at: 0001-01-01T00:00:00Z"
+
+
+  # Create a storage pool, create a volume snapshot and set its expiration timestamp
+
+  # shellcheck disable=2039,3043
+  local LXD_STORAGE_DIR lxd_backend
+
+  lxd_backend=$(storage_backend "$LXD_DIR")
```

Comment:
I think a default pool for the active storage driver is created by default for all tests, so I dont see why we need to create one here.